launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -95,7 +95,8 @@ public boolean isFinal() {
   void kill();
 
   /**
-   * Disconnects the handle from the application, without stopping it. After this method is called,
+   * Disconnects the handle from the application. If using {@link SparkLauncher#autoShutdown()}
```

Comment:
Sorry, I thought it was Scala one. I just checked it passes the doc build locally with the suggestion above.